### DISCUSSION TITLE
Workspace Admin Screen

### DISCUSSION THREAD

**Discussion by AdventistVirtual** [mckaywrigley/chatbot-ui#1268] [Ideas]
Thoughts on adding a new Workspace Admin Screen? A place that could allow admins to:

- Workspace Overview: Easily view and manage all workspaces. It could also serve as a centralized dashboard for monitoring users, activity, usage stats, etc.  This might also be useful for a "platform admin" to see workspaces and usage platform wide.
- Member and Role Management: Add the ability to add people to workspaces and assigning roles (Admin vs Member).
- Assistant Integration: Add the ability to assign global assistants to workspaces, giving accessible to all members.

**Comment by solaius** [comment]
Would love to have an admin screen good idea.  
Would add ability to manage and load prompts into prompt library for users or prompts that load for custom assistants.
